### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by JustoRago**
Added contact form and footer:

-Created contact form and styled it
-Created footer with links

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -315,3 +315,97 @@ h4 {
...
+  position: relative;
+  bottom: 40px;
+}
+
+footer {
+  position: absolute;
+  top: 4685px;
+  left: 125px;
+  width: 100%;
+}
```
Comment (levy002) [inline styles.css]: - [x] Kindly fix your footer and place it correctly at the end of the page as it is on the provided figma. :wink:

  - ### Your design
  image


  - ### Figma design
  image

File: styles.css [hunk 2]
```diff
@@ -315,3 +315,97 @@ h4 {
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 40px;
+  display: flex;
+  justify-content: center;
+  color: #9490ff;
+  position: relative;
+  top: 50px;
+  align-self: center;
+}
```
Comment (levy002) [inline styles.css]: - [x] Kindly make sure to stick to the provided design by adjust our contact section content typography. :smiley: 

  - ### Your design
  image

   - ### Figma design
  image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @asdt560, 
### I'm Levy, your PR reviewer

Excellent, you did a great job :sunglasses:
However, they are some issues that you need to address before you move on to the next project :wink:

## Highlights
- Nice design :heavy_check_mark: 
- Good job for using the correct git-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 

### Status: REQUSTED CHANGES :recycle: 
_check comments under_
- [ ] Kindly let's make sure to add all contributors in README file. :wink:
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kamba56 — APPROVED** [review]
Hi @asdt560, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ClappingLeonardoDicaprioGIF

## Highlights
- [x] All linters passed
- [x] Correct use of GitHUb flow
- [x] Professional readme file

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**